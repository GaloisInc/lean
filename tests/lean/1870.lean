structure T := (x y : nat)
example : T := by refine { x := 1, y := _}
example : T := by refine { x := 1, ..}
example : T := { x := 1, ..}
