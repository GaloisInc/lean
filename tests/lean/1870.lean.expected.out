1870.lean:2:18: error: tactic failed, there are unsolved goals
state:
⊢ ℕ
1870.lean:3:18: error: tactic failed, there are unsolved goals
state:
⊢ ℕ
1870.lean:4:15: error: don't know how to synthesize placeholder
context:
⊢ ℕ
